packages/documentation/docs/controls/_date-input_code.mdx

```diff
@@ -1,12 +1,9 @@
 import Playground from '@site/src/components/PlaygroundV2';
 import Props from '@site/docs/auto-generated/ix-date-input/props.md';
 import Events from '@site/docs/auto-generated/ix-date-input/events.md';
-import Tags from '@site/docs/auto-generated/ix-date-input/tags.md';
 
 # Date input
```

Comment:
Should we also remove the heading from the code file like done in the image below?

![image](https://github.com/user-attachments/assets/6f5fb118-d89f-4441-a0c1-98f00387a950)